serveradmin/servershell/helper/__init__.py

```diff
@@ -9,7 +9,8 @@ def get_default_shown_attributes():
 
     shown_attributes = list(Attribute.specials.keys())
     shown_attributes.remove('object_id')
-    shown_attributes.append('state')
+    shown_attributes.extend([attr.attribute_id for attr in
+                             ServertypeAttribute.objects.filter(default_visible=True).only('attribute_id').distinct()])
```

Comment:
To have a better readability I would suggest to store this line in an own variable like `default_attributes` or the like.